vmm/src/lib.rs

```diff
@@ -1192,6 +1225,19 @@ impl Vmm {
             .load_kernel()
             .map_err(|e| VmmActionError::StartMicrovm(ErrorKind::Internal, e))?;
 
+        // This is x86_64 specific since on aarch64 the commandline will be specified through the FDT.
+        #[cfg(target_arch = "x86_64")]
+        self.load_kernel_cmdline().map_err(|e| {
```

Comment:
For situations like this, I think it should suffice to open an issue and solve the problem for all cases in a separate PR.

Reply:
Yes, ignore this `.map_err( match e {...} )`, it goes away completely in https://github.com/firecracker-microvm/firecracker/pull/1063